phi/tools/video.py

```diff
@@ -0,0 +1,132 @@
+from phi.tools import Toolkit
+from moviepy import VideoFileClip, TextClip, CompositeVideoClip  # type: ignore
+from openai import OpenAI
+
+client = OpenAI()
+
+
+class VideoTools(Toolkit):
```

Comment:
Maybe we should name it more explicitly, i.e. something like `MoviePyVideoTools` or `GenericVideoTools`

Reply:
done